praetorian_cli/handlers/run.py

```
@@ -175,35 +175,119 @@ def tool(sdk, tool_name, target, extra_config, credential, wait, use_agent, loca
+
+    try:
+        result = sdk.capabilities.list()
+        caps = result[0] if isinstance(result, tuple) else result
+        if not isinstance(caps, list):
+            caps = []
+    except Exception:
+        caps = []
+
+    if not caps:
+        click.echo('Could not fetch capabilities from backend.', err=True)
+        click.echo(f'\nLocally installable tools:')
```

Ruff (0.15.7)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)